protocol/signalfx/format/log/signalfx_log.proto

```diff
@@ -11,7 +11,7 @@ message LogRequest {
 
 // ResourceLogs handles either sharing a resource with list of LogRecord or unique for each logRecord
 message ResourceLogs {
-    Resource resource = 1;
```

Comment:
The reason we do this is because we probably going to change a bit the Resource definition by adding new fields.